File: Cosmos-IOS/Cosmostation/Cosmostation/Dao/Password.swift
//
//  Password.swift
//  Cosmostation
//
//  Created by yongjoo on 20/03/2019.
//  Copyright © 2019 wannabit. All rights reserved.
//

import Foundation

public class Password {
    var password_id: Int64 = -1;
    var password_resource: String = "";
    
    
    init(_ id:Int64, _ resource:String) {
        self.password_id = id;
        self.password_resource = resource;
    }
}

File: app/src/main/java/wannabit/io/cosmostaion/dao/Password.java
package wannabit.io.cosmostaion.dao;

public class Password {
    public String resource;
    public String spec;

    public Password() {
    }

    public Password(String resource) {
        this.resource = resource;
    }

    public Password(String resource, String spec) {
        this.resource = resource;
        this.spec = spec;
    }
}
